src/yang/ietf-tcp.yang

```diff
@@ -170,25 +171,27 @@ module ietf-tcp {
         leaf local-address {
           type inet:ip-address;
           description
-            "Local address that forms the connection identifier.";
+            "Identifies the address that is used by the local
+             endpoint for the connection.";
         }
 
         leaf remote-address {
           type inet:ip-address;
           description
-            "Remote address that forms the connection identifier.";
+            "Identifies the address that is used by the remote
+             endpoint for the connection.";
         }
 
         leaf local-port {
           type inet:port-number;
           description
-            "Local TCP port that forms the connection identifier.";
+            "Local TCP port used for the connection.";
         }
 
         leaf remote-port {
           type inet:port-number;
           description
-            "Remote TCP port that forms the connection identifier.";
+            "Remote TCP port used for the connection.";
```

Comment:
How about this?

"Identifies the remote TCP port used for the connection, and is one of the four elements that form the connection identifier."

Reply:
looks good to me.

Reply:
Fixed